scripts/pipelines/setup_env

```diff
@@ -12,7 +15,6 @@ git config --global user.email "noreply@acquia.com"
 mysql -u root -proot -e "CREATE DATABASE IF NOT EXISTS drupal"
 
 # Setup NVM.
-nvm install v8.1.4
-npm install -g npm
+nvm install stable --latest-npm
```

Comment:
Is this normal? Just going to the latest instead of specifying which version?

Reply:
Yeah I don't know what's optimal here. If we pick a specific version, we have to continually update this file like we're doing now, deal with potentially insecure versions getting installed, and deal with people complaining that it's not the version _they_ happen to want that day.

If we don't pick a specific version, it'll change when a new release comes out which could surprise people and break builds. Although for minor/patch releases, that's not likely to happen often.

Do you have a strong opinion? I guess a compromise would be to at least pin the major version: `nvm install 12 --latest-npm`

Reply:
Actually, I propose that we just take this NPM out of BLT's internal scripts, and move it into our acquia-pipelines.yml template. That way people can manage it themselves, or get rid of it entirely if they don't use node. Nothing in BLT itself uses node, so there's no reason for BLT to manage this.

Reply:
That sound good! That's the place for it